Libraries/NewAppScreen/components/Header.js

```diff
@@ -11,39 +11,38 @@
 'use strict';
 
 import React from 'react';
-import {View, Text, StyleSheet, ImageBackground} from 'react-native';
+import {Text, StyleSheet, ImageBackground} from 'react-native';
 import Colors from './Colors';
 
 const Header = () => (
-  <View style={styles.container}>
-    <ImageBackground
-      accessibilityRole={'image'}
-      source={require('./logo.png')}
-      style={styles.backgroundLogo}
-    />
-
+  <ImageBackground
+    accessibilityRole={'image'}
+    source={require('./logo.png')}
+    style={styles.background}
+    imageStyle={styles.logo}>
     <Text style={styles.text}>Welcome to React Native</Text>
-  </View>
+  </ImageBackground>
 );
 
 const styles = StyleSheet.create({
-  container: {
-    alignItems: 'center',
-    justifyContent: 'center',
-    paddingTop: 100,
+  background: {
     paddingBottom: 40,
+    paddingTop: 96,
     paddingHorizontal: 32,
     backgroundColor: Colors.lighter,
   },
-  backgroundLogo: {
-    position: 'absolute',
-    top: -20,
-    left: -200,
+  logo: {
     opacity: 0.2,
-    alignItems: 'center',
-    justifyContent: 'center',
-    height: 540,
-    width: 540,
+    overflow: 'visible',
+    resizeMode: 'cover',
+    /*
```

Comment:
I had a rough time trying to make the *full* React logo be nice and offset like in the designs.

Another thought I had here is to use a 9-patch-like rectangular image for the background, where the React logo is already cropped, and then fix it to the bottom left corner of this component, to ensure a consistent background image across platforms.